### PR TITLE
Add Pocket Shannon Alpha TestNet #3

### PR DESCRIPTION
**Pull request by Olshansk**
Please note that the ChainId  is `poktroll` for this version of the testnet.

---

The announcement for this TestNet went out last week: https://medium.com/decentralized-infrastructure/pocket-network-shannon-update-alpha-testnet-3-eca539a9e111

Copy-pasting the links for reference:
* 🪙 Token Faucet: [faucet.testnet.pokt.network](https://faucet.testnet.pokt.network/)
* 🗺️ Explorer: [shannon.testnet.pokt.network/poktroll/block](https://shannon.testnet.pokt.network/poktroll/block)
* 🏗️ Deploy your own gateway & Supplier: [pokt-network/poktroll-docker-compose-example](https://github.com/pokt-network/poktroll-docker-compose-example)
* 🍝 Copy-pasta your way to deploying on a Debian server: [pokt-network/poktroll-docker-compose-example/debian_cheatsheet.md](https://github.com/pokt-network/poktroll-docker-compose-example/blob/main/debian_cheatsheet.md)
* 🧑‍💻 Developer Onboarding: [dev.poktroll.com/develop/developer_guide/quickstart](https://dev.poktroll.com/develop/developer_guide/quickstart)
* 💽 Full Node Indexer: [pokt-network/pocketdex/](https://github.com/pokt-network/pocketdex/)
* 📖 General Documentation: [docs.pokt.network/pokt-protocol/the-shannon-upgrade](https://docs.pokt.network/pokt-protocol/the-shannon-upgrade)
* 📒 Technical Documentation: [dev.poktroll.com/](https://dev.poktroll.com/)
* 🧑‍💻 Shannon SDK: [pokt-network/shannon-sdk](https://github.com/pokt-network/shannon-sdk)

### CODE REVIEW

**Comment by Olshansk** [comment]
@blacklodge-admin Do we need to ping someone to get these in?

**Review by editaahn — COMMENTED** [review]
Hi! Could you check the validations ran by Github Action? You can test code on your local environment, though

File: images/pokt-testnet/chain.png
Comment (editaahn) [inline images/pokt-testnet/chain.png:1]: The image directory should be `poktroll`.

**Comment by editaahn** [comment]
Please remove the coinGeckoId as it's a testnet chain information and may cause issues when the mainnet is released.

File: cosmos/poktroll.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "chainId": "poktroll",
+  "chainName": "Pocket Network",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pokt-testnet/chain.png",
```
Comment (editaahn) [inline cosmos/poktroll.json:4]:
```suggestion
  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/poktroll/chain.png",
```

**Comment by Olshansk** [comment]
@editaahn Do you have a quick idea of how to get this over the finish line?

```bash
Run for file in cosmos/akashnet.json cosmos/kaiyo.json cosmos/loop.json cosmos/poktroll.json cosmos/self-dev.json cosmos/shentu-2.2.json cosmos/yulei-2.1.json evm/eip155:14.json; do
  for file in cosmos/akashnet.json cosmos/kaiyo.json cosmos/loop.json cosmos/poktroll.json cosmos/self-dev.json cosmos/shentu-[2](https://github.com/chainapsis/keplr-chain-registry/actions/runs/10607825585/job/29414430411?pr=784#step:7:2).2.json cosmos/yulei-2.1.json evm/eip155:14.json; do
    if [[ $file == *".json"* ]]; then
      yarn ts-node src/index.ts $file
    fi
  done
  shell: /usr/bin/bash -e {0}
Failed to fetch coinGeckoId certik
Error: Process completed with exit code 1.
```

**Comment by editaahn** [comment]
I'm not sure why that message occurs. Could you try running the failed job again or pulling the current changes?

**Comment by Olshansk** [comment]
@editaahn Done. Let's see if its 🟢

**Comment by editaahn** [comment]
Thank you for contribution!

**Comment by Olshansk** [comment]
@editaahn Thanks for the support!

I was wondering where in the keplr extension could I view this now?

**Pull request merged**